src/forwarder.ts

```diff
@@ -63,6 +79,43 @@ async function validateForwarderArn(aws: Aws, functionArn: CloudFormationObjectA
   }
 }
 
+export async function addExecutionLogGroupsAndSubscriptions(
+  service: Service,
+  aws: Aws,
+  functionArn: CloudFormationObjectArn | string,
+) {
+  const extendedProvider = (service.provider as any)?.logs;
+
+  if (!isLogsConfig(extendedProvider)) {
+    return;
+  }
+
+  const resources = service.provider.compiledCloudFormationTemplate?.Resources;
+  if (restExecutionLoggingIsEnabled(extendedProvider)) {
+    // create log group
+    const logGroupName = await createRestExecutionLogGroupName(aws);
+    const executionLogGroupKey = "RestExecutionLogGroup";
+    const executionLogGroupName = addExecutionLogGroup(logGroupName);
+    resources[executionLogGroupKey] = executionLogGroupName;
+    // add subscription
+    const executionSubscription = subscribeToExecutionLogGroup(functionArn, executionLogGroupKey);
+    const executionSubscriptionKey = "RestExecutionLogGroupSubscription";
+    resources[executionSubscriptionKey] = executionSubscription;
+  }
```

Comment:
It looks like `executionLogGroupKey` and `executionSubscriptionKey` are constants -- I would suggest moving those up outside of the function since they stay the same every time you call the function. Also, I generally follow the convention of making constants in TypeScript/JS `ALL_CAPS_SNAKE_CASE`.

Note that when I say constant I don't mean anything that uses `const` -- that just means it can't be reassigned to something else. I mean constant in the sense of some kind of string or number with a special meaning.